Makefile

```diff
@@ -11,6 +11,7 @@ test:
 	$(PYTHON) -m pytest --cov=trio_websocket --no-cov-on-fail
 
 lint:
+	$(PYTHON) -m black trio_websocket/ tests/ autobahn/ examples/
```

Comment:
Hi @jakkdl 

> I was requested to add type hints to this repository

Would you share some context of that conversation?  (gitter link etc.)

Please see the thread here before we discuss type annotation coverage.
https://github.com/python-trio/trio-websocket/pull/167#discussion_r780850077

> > Would it be reasonable to revisit if the project becomes more actively maintained?
> 
> is now relevant

To me, being off life-support maintenance would mean things like 1) there is at least another person that has some history on the project and I can trust to share the load on whatever comes up going forward,  2) there is activity like revising the API, rewriting the implementation or tests, etc.  I wouldn't say that adding type annotations or auto formatting means that the project is out of the woods regarding stewardship, and there are aspects of full type annotations that make maintenance harder.

Regards,

Reply:
@Zac-HD

Reply:
Thank you! We'll look into possible alternative libraries that are under more active support and reconsider whether to use trio-websockets. If we don't find any great alternatives, I could go with a more minimal approach and just write separate stub files that needn't touch the existing code at all, and if you don't want the extra maintenance burden of them we can just stick them in an entirely separate repo.